src/aics-image-viewer/components/AxisClipSliders/styles.css

```diff
@@ -89,7 +102,7 @@
   &.clip-sliders-2d {
     /* Make room for play buttons */
     /* .axis-slider (480) + .slider-name (10) + .slider-slices (70) + .slider-play-buttons (95) + margins & padding (100) */
-    max-width: 755px;
+    max-width: 805px;
```

Comment:
I think the comment above this line needs an update to say where the value came from